packages/tokens/README.md

```diff
@@ -0,0 +1,81 @@
+# @openfun/cunningham-tokens
+
+## Introduction
+
+**What are design tokens ?**
+
+Design tokens are the fundamental variables defining the precise behavior and rendering of ui components.
+
+For example:
+- The primary color of a text element
+- The standard spacing between two elements
+- The border radius of a button
+- ...
+
+## Getting started
+
+In this section we will install the library and generate the file that contains the design tokens of your app in order to 
+make Cunningham's Design System yours!
+
+Install the lib
+
+```
+yarn add @openfun/cunningham-tokens
+```
+
+Create a file named `cunningham.cjs` at the root of your project
+
+```
+module.exports = {
+  theme: {
+    colors: {
+      primary: 'purple'
+    },
+  },
+};
+```
+
+In this configuration file you can overwrite all the default values of the design system. 
+You can find the default values [here](./src/bin/cunningham.dist.js).
+
+Now add this script to your `package.json`
+
+```
+{
+    "scripts": {
+        ...
+        "build-theme": "cunningham"
+    }
+}
+```
+
+And in order to generate the tokens css file, run 
+
+```
+yarn build-theme
+```
+
+It will generate a file named `cunningham-tokens.css`. **Don't forget to run this command everytime you
+change the content of the `cunningham.cjs` file !**
```

Comment:
Less confusing what do you think ?
```suggestion
change the content of the `cunningham.js` file !**
```

Reply:
I would prefer toop, however, this is not possible to do so because most of frontend's package will have `type: "module"` in their `package.json` and because of Node.js CJS constraints that we previously talked about we **must** import config file a CJS, not ESM. 

So shortly: if the extension is .js it will try to import the config file as ESM within a CJS executed bin, and it will crash 😅
Using the `cjs` extension forces Node.js to treat it as CJS even if it is in a module setup.

Reply:
For instance you can take a look to how Storybook generated its config file in the react repo. It did the same thing.

Reply:
Indeed ! ESMess